### DISCUSSION TITLE
Data grid

### DISCUSSION THREAD

**Discussion by Sath-Satya** [Taiizor/ReaLTaiizor#41] [General]
Thanks for your work. Really appreciate it. 
How I can obtain datagrid from crown toolsets

**Comment by Taiizor** [comment]
The Crown theme does not have DataGrid.

**Reply by Stefan6440** [reply]
I know this might come to late :-)   Creating an complete datagrid is a huge task.  I have used MaterialListView with pretty good results. You will though have to handle crud functionality and some ownerdrawing by your self, but thats minimal compared to the timesaving using the listview.
